NYTimes.com.js

```diff
@@ -90,14 +94,46 @@ function scrape(doc, url) {
 			}
 		}
 		item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
-		var pdfurl = ZU.xpathText(doc, '//div[@id="articleAccess"]//span[@class="downloadPDF"]/a[contains(@href, "/pdf")]/@href');
+		item.libraryCatalog = "NYTimes.com";
+		// Convert title/tags of NYT archives when in all caps: 
+		if (item.title == item.title.toUpperCase())
+				item.title = ZU.capitalizeTitle(item.title, true);
+		var allcaps = false;
+		for (i=0; i < item.tags.length; i++) {
+			if (item.tags[i] == item.tags[i].toUpperCase())
+				allcaps = true;
```

Comment:
ah yes, good point. Then what you suggest makes sense. Start with `allcaps = true` and then set it to false if there's one case where the tag is not identical to the uppercased one. You should then add `break;` so you don't unnecessarily run through all other tags.